### PR TITLE
Remove Board's notifications about pending comps

### PR DESCRIPTION
**Pull request by pedrosino**

### CODE REVIEW

**Comment by jfly** [comment]
Looks like some tests need updating, but the code LGTM!

**Comment by AlbertoPdRF** [comment]
Tests are passing now! Thanks @pedrosino

File: WcaOnRails/spec/helpers/notifications_helper_spec.rb [hunk 1]
```diff
@@ -79,27 +79,28 @@
...
-          {
-            text: "#{visible_unconfirmed_competition.name} is visible, but unlocked",
-            url: admin_edit_competition_path(visible_unconfirmed_competition),
-          },
-        ]
-      end
-    end
+    # Make a test for WCAT instead
+    # context "when signed in as a board member" do
+    #   let(:board_member) { FactoryBot.create :user, :board_member, :wca_id }
```
Comment (viroulep) [inline WcaOnRails/spec/helpers/notifications_helper_spec.rb]: Maybe you can keep the same test, but replace `board_member` by [`wcat_member`](https://github.com/thewca/worldcubeassociation.org/blob/master/WcaOnRails/spec/factories/users.rb#L79)?

Comment (pedrosino) [reply WcaOnRails/spec/helpers/notifications_helper_spec.rb]: Ah, of course. I searched for such a factory but forgot we use the short names :P

File: WcaOnRails/spec/helpers/notifications_helper_spec.rb [hunk 2]
```diff
@@ -79,8 +79,8 @@
       end
     end
 
-    context "when signed in as a board member" do
-      let(:board_member) { FactoryBot.create :user, :board_member, :wca_id }
+    context "when signed in as a WCAT member" do
+      let(:wcat_member) { FactoryBot.create :user, :wcat_member, :wca_id }
```
Comment (viroulep) [inline WcaOnRails/spec/helpers/notifications_helper_spec.rb]: @pedrosino You forgot to update the `board_member` variable below, which raised the error in the tests.

**Pull request merged**